content/docs/concepts/accountability/_index.md

```diff
@@ -0,0 +1,385 @@
+- `slashing rate` is computed:
+  - `base rate + epoch offences count * collusion factor + history * history factor`.
+- `slashing amount` of the fine is computed:
+  - `(slashing rate * validator bonded stake)/slashing rate precision`
+- the slashing is computed:
+  - the slashed amount of NTN stake token is subtracted from the validator’s bonded stake and transferred to the Autonity Protocol global `treasury` account for community funding
+  - the slashing fine is applied to validator bonded stake according to the protocol’s [Penalty Absorbing Stake (PAS)](/concepts/accountability/#penalty-absorbing-stake-pas) model
+  - the `jail period` of the validator is computed to determine the validator's jail release block number: `current block number + jail factor * history * epoch period`.
+- the validator state is updated: (a) the self-bonded and total staked amounts, (b) the slashing amount is added to the validator's `totalSlashed` amount.
+
+### Penalty Absorbing Stake (PAS) 
+
+Autonity implements a [_penalty absorbing stake_](glossary/#penalty-absorbing-stake-pas) model where validator [self-bonded](glossary/#self-bonded) stake is slashed in priority to [delegated](glossary/#delegated) stake when applying slashing penalties for accountability events.
+
+Slashing priority is simply:
+
+- self-bonded stake is slashed as first priority until exhausted
+- delegated stake is slashed as second priority when the slashing amount exceeds the amount of self-bonded stake available.
+
+Consequently, validator operator stake ("skin in the game") is at first risk and provides loss absorbing capital in the case of a slashing event. Self-bonding stake is not just a source of income but a public commitment to the operational integrity of a validator node by a validator operator. Potential stake delegators can use the amount of self-bonded stake of a validator as a decision factor when conducting due diligence before staking.
+
+## Rules
+
+### Accountability rules
+
+Accountability rules are applied to detect faults in the three Tendermint consensus round phases *propose*, *prevote*, *precommit*. Rules are detailed in the table below.
+
+{{% alert title="Note" %}}
+In the table:
+
+- *value*: is shorthand for "block proposal"
+- Rule ID: is the unique identifier for the Rule defined in the AFD rule engine. ID prefixes correspond to Tendermint consensus phases:
+  - `PO`: *propose*
+  - `PV`: *prevote*
+  - `C`: *precommit*
+{{% /alert %}}
+
+| Rule ID | Description |
+| --| --|
+| `PN` | Proposer has proposed a new *value* (i.e. block proposal) having (a) already sent a precommit message earlier, or (b) has received sufficiently many prevotes for an earlier proposal *value* during the same round. |
+| `PO` | Proposer has proposed a *value* (i.e. block proposal) that has already been proposed without attaching as justification the `2f + 1` prevotes from the same round for the value. |
```

Comment:
Theoretically this is correct, but the behaviour in the code is the following. 

When you see a old proposal with validRound = vr you check:
1. Did the proposer precommit on a value different than the one he proposed in vr? If so --> misbehaviour
2. did he precommit for something different then nil in the rounds between vr and current round? if so ---> misbeahviour
3. is there a quorum of prevotes  in vr for a value different than the proposed one ? --> misbehaviour
4. If we do not see any quorum in vr--> accuse proposer, he will have to send the quorum of prevotes to prove its innocence

Now the question that comes to my mind is: what do we want to accomplish with this rule section?

Do we want to give a complete albeit concise description of what is checked in the code, or do we just want to give a sense to the reader of what this misbehaviour means?

Reply:
Good question:

>Now the question that comes to my mind is: what do we want to accomplish with this rule section?

The thinking here is to anticipate and answer the reader question of "OK, so there is accountability and fault detection for committee members that break the consensus rules. So, (a) what are the rules?, and, (b) what does that rule mean?

>Do we want to give a complete albeit concise description of what is checked in the code

No, that seems too low-level.

>, or do we just want to give a feeling to the reader of what this misbehaviour means?

Yes, absolute. So, I thought a simple listing of Rule ID + Description would answer that.

Reply:
Ok then I will re-adjust my comments with this in mind.

Reply:
Terminology simplified in commit [d752ba8](https://github.com/autonity/docs.autonity.org/pull/81/commits/d752ba822288994db03315f7b67b6e9453331bc1): *value* replaced with simple 'block proposal'

Reply:
The proposer does not have to attach `2f + 1` prevotes to an old proposal.

Personally I would just say something along the lines of:
- PN, PO --> you did not respect the tendermint rules related to proposal
- PVN, PVO, ... --> you did not respect the tendermint rules related to prevote
- C --> ... precommit

In my opinion if we want to explain these rules 1 by 1 we are basically re-writing the D3 paper. I think it would make more sense to give just a high-level overview of the rules, and link to the paper for more details.

Reply:
@lorenzo-dev1 the rule table has been edited to simplify on the principle of simply having the rule id and a simple '1-liner' high level description'. 

So, the reader can then get a high-level understanding of the penalty being slashed for. For more detail, they can drill into the code.

Reply:
Ok, I will resolve this thread and open new ones if I have comments on the signle rules explaination.